neptunecontrib/monitoring/skopt.py

```diff
@@ -62,7 +74,7 @@ def _get_last_params(res):
         return str(named_params)
 
 
-def log_results(results, experiment=None):
+def log_results(results, experiment=None, log_plots=True, log_pickle=True):
     """Logs runs results and parameters to neptune.
 
     Logs all hyperparameter optimization results to Neptune. Those include best score ('best_score' channel),
```

Comment:
The `diagnostics` channel doesn't exist in this implementation any more.
Also, we call them "logs" these days :)

Reply:
Mhm, will revert to that.

Reply:
Done